go.sum

```diff
@@ -8,6 +8,8 @@ github.com/elastic/go-ucfg v0.8.4-0.20200415140258-1232bd4774a6 h1:Ehbr7du4rSSEy
 github.com/elastic/go-ucfg v0.8.4-0.20200415140258-1232bd4774a6/go.mod h1:iaiY0NBIYeasNgycLyTvhJftQlQEUO2hpF+FX0JKxzo=
 github.com/elastic/package-registry v0.7.0 h1:32fAVDQNNJ+mE/8zVNXfUCPX99Z6DLghmPkI4YZBhRA=
 github.com/elastic/package-registry v0.7.0/go.mod h1:ERTTIxAsQOCVZJDqR4LJbDDAtxV+pz4wdPPrKheiAUc=
+github.com/elastic/package-registry v0.9.0 h1:bfVF5k6w43zjYaF+0t3fRCU1IguRhwqeY1wjG1LMA6k=
+github.com/elastic/package-registry v0.9.0/go.mod h1:oQx3Tg9ynuC6APd0o0OHud9kyPX6S6IzdJp/R4Hj1HY=
```

Comment:
Nit: run `go mod tidy` to clean up older versions of dependencies in this file?

Reply:
Thanks!